CLAUDE.md

```
@@ -0,0 +1,115 @@
+make idea-discussion-containers-start
+```
+
+---
+
+## アーキテクチャ
+
+### モノレポ構成
+
+npm workspaces を使用。ルートの `package.json` が 3 つのワークスペースを定義：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)